client/lib/feature-detection/index.js

```diff
@@ -0,0 +1,19 @@
+/** @format */
+
+const featureDetection = {
```

Comment:
What do you think to exporting each `supportsX` individually?

I believe that this would allow for imports to be a little more efficient as we're not passing around a potentially large object (as this lib grows). Instead, chunked bundles would ship with just the checks they need.

Right now this lib is small, so there's not a real need to do that so I'm onboard if you'd rather this be tackled in the future as the need arises :)